src/app/lists/ListDataview.tsx

```diff
@@ -56,13 +58,39 @@ export const headerKeysAndLabels = [
     ["10", "Family of 10"],
 ] satisfies [keyof ListRow, string][];
 
+export const listRowToListDB = (listRow: ListRow): Schema["lists"] => ({
+    item_name: listRow.itemName,
+    notes_for_1: listRow[1]?.notes,
```

Comment:
i don't think you need the ? anymore
```suggestion
    notes_for_1: listRow[1].notes,
```